ruby_exercises/mythical-creatures/spec/werewolf_spec.rb

```diff
@@ -66,19 +72,43 @@ def initialize
   end
 
   it 'consumes a victim' do
-    # your code here
+    werewolf = Werewolf.new('David', 'London')
+    victim = Victim.new
+
+    werewolf.change!
+    werewolf.consume(victim)
+
+    expect(werewolf.victims).to eq ([victim])
   end
 
   it 'cannot consume a victim if it is in human form' do
-    # your code here
+    werewolf = Werewolf.new('David', 'London')
+    victim = Victim.new
+
+    werewolf.consume(victim)
+
+    expect(victim.status).to eq (:alive)
```

Comment:
I would add another expect here that shows that `werewolf.victims` is an empty array.